### DISCUSSION TITLE
Can ArchR be used to analyze scNOMe-seq data?

### DISCUSSION THREAD

**Discussion by jiaqiqi77** [GreenleafLab/ArchR#1432] [Questions / Documentation]
Hello, I know that archr is very powerful when applied to peaks obtained from atac-seq data, I would like to ask if it can be applied to open region NDR (nucleosome-depleted regions) obtained from scNOMe-seq, similar to peaks in atac-seq data ? In addition, if possible, starting from the length of the atac-seq peak, are the methods in archr applicable to the analysis of ndr?

**Comment by rcorces** [comment]
I have never worked with NOMe-seq data so I'm not really able to answer this question. There are basic assumptions made in ArchR that may not apply to other types of data and the tool is not designed to analyze other types of data so we cant provide support for those applications. If you use ArchR and find that it works well, it would be great if you report back with your experience.
